ogr/services/abstract.py

```diff
@@ -95,6 +127,18 @@ def get_pr_comments(self, pr_id):
         """
         raise NotImplementedError()
 
+    def pr_create(self, title, body, target_branch, current_branch):
```

Comment:
how about target remote? I may suggest to rename current_branch to source_branch

Reply:
> target remote

Since we are working with `upstream repo` <-> `fork` relation, we do not need to know it.

- When we have PR on the upstream project, the target remote is the repo itself.
- When we have forked project, we are creating PR against parent project.

I'm not sure about the multiple-remote situations. It can be hard to make it compatible between different services.

Reply:
I didn't realize, sounds good then!